Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM python:3
 WORKDIR /home/app
 
 #If we add the requirements and install dependencies first, docker can use cache if requirements don't change
-ADD /backend/requirements.txt /home/app
-RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
Can we leave the `--no-cache-dir` or is there a reason we are removing it?

Reply:
I added it back in.